shell/common/engine.cc

```diff
@@ -314,10 +316,12 @@ bool Engine::HandleLifecyclePlatformMessage(PlatformMessage* message) {
       state == "AppLifecycleState.detached") {
     activity_running_ = false;
     StopAnimator();
+    is_backgrounded_sync_switch_->SetSwitch(true);
   } else if (state == "AppLifecycleState.resumed" ||
              state == "AppLifecycleState.inactive") {
     activity_running_ = true;
     StartAnimatorIfPossible();
+    is_backgrounded_sync_switch_->SetSwitch(false);
```

Comment:
This is less problematic but still would probably be better to just let the embedders who care about this set it rather than doing it for all of them on these increasingly overloaded enum values.

Reply:
SGTM as per previous comment.